README.md

```diff
@@ -426,6 +501,124 @@ roles=User,Admin,SuperAdmin
 ```
 </details>
 
+### @Scope
+
+---
+**[@Scope](#scope)** allow to specify `Singleton` or `Prototype` for bean definition.
+
+`Singleton` - returns single instance of class for every **[Bean](#bean)**
+
+`Prototype` - returns a new instance of class for every **[Bean](#bean)**
+
+The annotation could be specify on the class level if the class marked as **[@Component](#component)**:
+
+<details>
+<summary>Example</summary> 
+
+```java
+@Component
+@Scope(BeanScope.PROTOTYPE)
+public class MessageService implements CustomService {
+
+    private String hello = "Hello";
+
+    public String getMessage() {
+        return hello;
+    }
+
+}
+```
+</details>
+
+The annotation could be specify on the method level if the method marked as **[@Bean](#bean)**:
+
+<details>
+<summary>Example</summary> 
+
+```java
+package com.bobocode.svydovets.autowiring.configuration;
+
+import com.bobocode.svydovets.annotation.annotations.Bean;
+import com.bobocode.svydovets.annotation.annotations.Configuration;
+import com.bobocode.svydovets.annotation.annotations.Scope;
+import com.bobocode.svydovets.annotation.register.BeanScope;
+
+@Configuration
+public class TestConfig {
+
+    @Bean
+    @Scope(BeanScope.PROTOTYPE)
+    public FooService fooService() {
+        FooService fooService = new FooService();
+        fooService.setMessage("Foo");
+        return fooService;
+    }
+
+}
+```
+</details>
+
+### @Value
```

Comment:
I created the branch from @sleepnotdead no idea why it still shows diffs when original brach is merged